examples/storage_demo.yaml

```diff
@@ -40,7 +37,7 @@ resources:
 # 3. You want to have a write-able path to directly write files to S3 buckets -
 #    specify a name (to create a bucket if it doesn't exist) and set the mode
 #    to MOUNT. This is useful for writing code outputs, such as checkpoints or
-#    logs directly to a S3 bucket.
+#    logs directly to a S3/GCS bucket.
```

Comment:
Inconsistency, lets keep s3 and gcs lower case?

Reply:
hmm in this doc, we consistently use `S3/GCS` when referring to the services, and lowercase `s3/gcs` when used in urls/paths

Reply:
Sounds good!